src/lucky/paginator/components/bulma_nav.cr

```diff
@@ -1,9 +1,12 @@
+require "../../html/base_component"
+
 # Pagination component using Bulma Pagination styles
 #
 # https://bulma.io/documentation/components/pagination/
 class Lucky::Paginator::BulmaNav < Lucky::BaseComponent
   needs pages : Lucky::Paginator
 
+  @[Deprecated("Consider using Lucky::Paginator::SimpleNav or writing a custom component")]
```

Comment:
Is it an idea to put those front-end framework-specific components in a separate shard? Or maybe just document them?

Reply:
I don't think so. I think just documenting them would be much better. Like recipes if you need. Then adding them into your project is easy and don't have to worry about outdated CSS framework versions.